### ISSUE TITLE
oembed broken in short description

### ISSUE THREAD

**Issue by BHEADRICK** [woocommerce/woocommerce#13625]
## EXPLANATION OF THE ISSUE

WC Version: 2.6.14
WP Version: 4.7.3
Storefront version: 2.1.8

Storefront theme active
No other plugins active

Adding a youtube url into the short description renders an oembed correctly on the edit screen in visual mode, but when viewing the on the frontend, the oembed does not render, and only the url is displayed

Oembeds work correctly in the main content on products and other post types, just not the short description on products. 
## STEPS TO REPRODUCE THE ISSUE

copy youtube url into short description
(see that the oembed works correctly in the editor)
either click preview or click update and then view the product
notice that the oembed does not work. 

## SYSTEM STATUS REPORT

```
Grab the system status report from WooCommerce > System Status and paste it here.
```
`
### WordPress Environment ###

Home URL: https://poolwarehouse.dev
Site URL: https://poolwarehouse.dev
WC Version: 2.6.14
Log Directory Writable: ✔
WP Version: 4.7.3
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US

### Server Environment ###

Server Info: nginx/1.11.3
PHP Version: 7.0.10-1+deb.sury.org~trusty+1
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.35.0
OpenSSL/1.0.1f

SUHOSIN Installed: –
MySQL Version: ❌ 5.5.50 - We recommend a minimum MySQL version of 5.6. See: WordPress Requirements
Max Upload Size: 1 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 2.6.14
: 
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔

### Active Plugins (2) ###

WooCommerce: by WooThemes – 2.6.14
WP Security Audit Log: by WP White Security – 2.6.1

### Settings ###

Force SSL: ✔
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2

### API ###

API Enabled: ✔

### WC Pages ###

Shop Base: #9 - /shop/
Cart: #10 - /pool-kit-shopping-cart/
Checkout: #11 - /pool-kit-checkout/
My Account: #13 - /my-account/

### Taxonomies ###

Product Types: bundle (bundle)
external (external)
grouped (grouped)
simple (simple)
variable (variable)

### Theme ###

Name: Storefront
Version: 2.1.8
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you're modifying WooCommerce on a parent theme you didn't build personally
then we recommend using a child theme. See: How to create a child theme

WooCommerce Support: ✔

### Templates ###

Overrides: –
`

**Comment by mikejolley** [comment]
@claudiulodro want this one? Short description may be missing a filter regular content has.

**Comment by claudiosanches** [comment]
@claudiulodro we should do it on `woocommerce_short_description`, but maybe will cause problems on REST API.

**Comment by claudiulodro** [comment]
I can definitely take a look at it.

**Comment by mikejolley** [comment]
This will be fixed in 3.1.
